client/src/Movies/Movie.js

```diff
@@ -11,7 +11,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.parms.id;
```

Comment:
Watch the spelling of your properties! You can always go to the react dev tools to see what you have on your props